guides/common/modules/ref_system-requirements.adoc

```diff
@@ -19,13 +19,13 @@ endif::[]
 ifdef::katello,satellite[]
 ifeval::["{context}" == "{project-context}"]
 * A minimum of 20 GB RAM is required for {ProjectServer} to function.
-In addition, a minimum of 4 GB RAM of swap space is also recommended.
+In addition, a minimum 4 GB RAM of swap space is also recommended.
```

Comment:
Was this change intentional? If not, please revert or move to a separate commit. As goes for below.

Reply:
Yes. This change is intentional from my side. Corrected from grammatical POV.

Reply:
Is it a correction? To me it reads worse.

And if it's a correction, it deserves its own PR. Adding multiple fixes in a single PR (especially one that's complicated) is a recipe for never extending the time to review by a lot.

Reply:
Alright. I will revert it. :+1:  (Thought it's small and we often correct it so I considered it here itself).

Reply:
> On a general note: I want to investigate if it's possible to omit the "project-context" because it should be identical to {build} as defined in guides/common/attributes.adoc. However, this should not be part of your PR.

For `katello` it loads fewer users (which is incorrect according to snip files). It is calling the file `snip_***foreman.adoc` instead `snip_***katello.adoc`

Reply:
> For now, I think we can safely remove the "ifeval" in each snippet because we're already including the correct snippet per build target.

Noted :+1:

Reply:
> For `katello` it loads fewer users (which is incorrect according to snip files). It is calling the file `snip_***foreman.adoc` instead `snip_***katello.adoc`

I am also concerned about this.